src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -419,9 +407,12 @@ def set_registration_options
         options = Storage::InstallationOptions.instance
         case action
         when :register_scc
-          options.email = Yast::UI.QueryWidget(:email, :Value)
-          options.reg_code = Yast::UI.QueryWidget(:reg_code, :Value)
+          options.email      = Yast::UI.QueryWidget(:email, :Value)
+          options.reg_code   = Yast::UI.QueryWidget(:reg_code, :Value)
+          options.custom_url = nil
         when :register_local
+          options.email      = "" # Use an empty string like InstallationOptions constructor
```

Comment:
NP: I'd prefer adding the comment above the code, trailing comments usually make the lines quite long...